test/Twilio.Test/Twilio.Test.csproj

```diff
@@ -2,28 +2,32 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <RootNamespace>Twilio.Tests</RootNamespace>
-    <TargetFrameworks>netcoreapp2.0;net451;net35</TargetFrameworks>
-    <RuntimeIdentifier Condition=" '$(TargetFramework)' == 'net451' ">win7-x86</RuntimeIdentifier>
+    <TargetFrameworks>net60;net462;net35</TargetFrameworks>
```

Comment:
Dropping unsupported netcoreapp2.0 and net451, replacing with supported net60 and net462.